source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -15,79 +15,153 @@
+          <li>
+            <span class="item">Proyectos:</span>
+            <!--ko if: isEvaluationEditable() && (userView == 1 || userView == 3) -->
+            <div class="editable-projects" data-bind="css: { edition: evaluation.project(), 'top-tip': evaluation.project()}, attr: {'data-tips': evaluation.project()}">
+              <input type="text" placeholder="Agregá los proyectos" data-bind="value: evaluation.project, event: {keyup: evaluation.onKeyUpProject, focusin: evaluation.onFocusInProject, blur: evaluation.onBlurProject}" /><span class="pencil"></span>
+            </div>
+            <!-- /ko -->
+            <!--ko if: !isEvaluationEditable() || (userView != 1 && userView != 3) -->
+            <span class="top-tip" data-bind="if: evaluation.project(), attr: { 'data-tips': evaluation.project()}"><span class="item-info" data-bind="text: evaluation.project()"></span></span>
+            <!-- /ko -->
+          </li>
+        </ul>
+        <h4>Métricas de Evaluación</h4>
+        <div class="metrics-container">
+          <span>1</span> No Alcanza las Expectativas - <span>2</span> Necesita Mejorar<br />
+          <span>3</span> Cumple con Expectativas - <span>4</span> Excede las Expectativas
+        </div>
+        <p class="explanatory-text">*Si no tenés conocimientos para calificar algún campo dejalo vacío</p>
+      </div>
+      <div class="top-right">
+        <!-- ko if: userView == 3 || evaluation.devolutionInProgress -->
+        <div>
+          <h4>Comentarios Generales</h4>
+          <div class="title">
+            <span>Fortalezas</span>
+            <div class="ck-button-edit" data-bind="if: isEvaluationEditable(), visible:!evaluation.isEditingStrengthsComment()" title="Editar">
+              <label>
+                <span><i></i></span>
+                <input type="checkbox" value="+" data-bind="checked: evaluation.isEditingStrengthsComment">
+              </label>
+            </div>
           </div>
-          <!-- /ko -->
-          <!--ko if: !isEvaluationEditable() || (userView != 1 && userView != 3) -->
-          <span class="top-tip" data-bind="if: evaluation.project(), attr: { 'data-tips': evaluation.project()}"><span class="item-info" data-bind="text: evaluation.project()"></span></span>
-          <!-- /ko -->
-        </li>
-      </ul>
-      <h4>Métricas de Evaluación</h4>
-      <div class="metrics-container">
-        <span>1</span> No Alcanza las Expectativas - <span>2</span> Necesita Mejorar<br />
-        <span>3</span> Cumple con Expectativas - <span>4</span> Excede las Expectativas
+          <div class="comments-container markdown-content" data-bind="markdownHtml:evaluation.strengthsComment, visible: !evaluation.isEditingStrengthsComment() && evaluation.strengthsComment()"></div>
+          <div class="content-editable" data-bind="visible: evaluation.isEditingStrengthsComment">
+            <textarea data-bind="value: evaluation.strengthsComment"></textarea>
```

Comment:
You said me that changes are accepted when the user ends editing, but since it is using the same observable I think that it is not working as you suppose. Please, review it to avoid possible misbehaviors.